aws-redshiftserverless-namespace/src/main/java/software/amazon/redshiftserverless/namespace/CreateHandler.java

```diff
@@ -14,26 +26,40 @@ public class CreateHandler extends BaseHandlerStd {
     private Logger logger;
 
     protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
-        final AmazonWebServicesClientProxy proxy,
-        final ResourceHandlerRequest<ResourceModel> request,
-        final CallbackContext callbackContext,
-        final ProxyClient<RedshiftServerlessClient> proxyClient,
-        final Logger logger) {
+            final AmazonWebServicesClientProxy proxy,
```

Comment:
can you run the formatter before submitting the PR if you haven't? looks like all of the existing formats are changed, I'd like to keep them as is, also we can keep the git history better like this